sormas-api/src/main/java/de/symeda/sormas/api/action/ActionFacade.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2018 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *******************************************************************************/
+package de.symeda.sormas.api.action;
+
+import java.util.Date;
+import java.util.List;
+
+import javax.ejb.Remote;
+
+@Remote
+public interface ActionFacade {
+
+	ActionDto saveAction(ActionDto dto);
+
+	ActionDto getByUuid(String uuid);
+
+	void deleteAction(ActionDto ActionDto);
+
+	List<ActionDto> getAllActiveActionsAfter(Date date);
```

Comment:
The "active" part can be removed from these method names since actions can't be archived and are fully deleted from the database when they're deleted (so there's no way for them to not be active)

Reply:
OK

Reply:
A precision, in fact, this filtered actions related to an archived context (event), but OK to rename the method.